recipes/iiboost/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "iiboost" %}
+{% set version = "0.2" %}
+{% set sha256 = "0ecd906b453a6a278651e324352bd6f820b70aa4ca15fa42843bdb5c01ee7079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/cbecker/iiboost/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    - cmath_fixes.diff
+    - python3.diff
+    - msvc_fixes.diff
+
+build:
+  number: 0
+  skip: True        # [np!=112 or (win and py27)]
+  features:
+    - vc14          # [win and py>=35]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - gcc                   # [linux]
+    - cmake
+    - eigen
+    - numpy x.x
+    - itk       4.11.*
+    - boost     1.63.*
```

Comment:
As noted above, these are being linked to by `iiboost` libraries directly. Hence why `install_name_tool` is fixing these linkages in `iiboost`'s libraries. Feel free to peruse the log if you are not convinced. The thing is if we don't list them here, it is going to be more error prone than if we do.

For example, say the `jpeg` version that `itk` uses changes. Will that break `iiboost`? FWICT based on these linkages yes. So we will need to rebuild `iiboost`. However if we haven't been tracking `jpeg` in `iiboost` previously, and we merely bump the build number on `iiboost`, how can we be sure which package of `itk`, `iiboost`, and `jpeg` we get during an install. If we get the old `itk` package pinned to the old `jpeg` and a newer `iiboost` package, then `iiboost` will be broken. Similarly if we have the newer `itk` and `jpeg`, but an older `iiboost`. The only way to safeguard against these problems is to pin `iiboost` dependencies within its own recipe.